main.py

```diff
@@ -2,6 +2,70 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.bulk_create(
+        [
+            Genre(name="Western"),
+            Genre(name="Action"),
+            Genre(name="Dramma")
+        ]
+    )
+
+    Actor.objects.bulk_create(
+        [
+            Actor(
+                first_name="George",
+                last_name="Klooney",
```

Comment:
The last name 'Klooney' seems to be a typo. It is later corrected to 'Clooney', but ensure that this is intentional and not a mistake in the initial creation.